templates/agent-conf.d/kubernetes.yaml.erb

```diff
@@ -0,0 +1,174 @@
+init_config:
+  # Initialization retries
+  #
+  # if the agent is expected to start before Kubelet,
+  # use these settings to configure the retry policy.
+  #
+  # init_retry_interval defines how long (in seconds) the kubelet client
+  # will wait before retrying initialization.
+  # Defaults to 0.
+  #
+  # init_retry_interval: 20
+  #
+  # init_retries configures how many retries are made before failing permanently.
+  # Defaults to 0.
+  #
+  # init_retries: 5
+
+instances:
```

Comment:
I don't think there's a use case where we'd want to support multiple instances for the `kubernetes` integration... Is that correct? Just double-checking in case you want to add the support for that now. It's no biggie, as it's fairly easy to add (maybe not in the most idiomatic way) while retaining backward compatibility, we can always add it in the future.

Reply:
Ill just leave that out for now then.